packages/@lwc/integration-karma/test/light-dom/slot-fowarding/slots/reactivity/index.spec.js

```diff
@@ -0,0 +1,124 @@
+import { createElement } from 'lwc';
+import { extractDataIds } from 'test-utils';
+
+import LightContainer from './x/lightContainer/lightContainer';
+
+describe('light DOM slot forwarding reactivity', () => {
```

Comment:
These tests ensure that dynamically setting the `slot` attribute on a forwarded slot works properly between light and shadow DOM.